tutorials/single-page-tutorials/nested-nfts-with-wallet-lib.md

```diff
@@ -0,0 +1,423 @@
+
+```
+node create-nfts.js
+```
+
+Since you were minting three NFTs, your console output should look like this:
+
+```console
+Start local IPFS node for upload:
+<IPFS-node-swarm-information>
+
+
+Your file "avatar.jpg" was successfully uploaded to IPFS:
+https://ipfs.io/ipfs/<ipfs-content-identifier>
+
+Your NFT was successfully minted in this block:
+https://explorer.shimmer.network/testnet/block/<blockId>
+
+
+
+Your file "mask.jpg" was successfully uploaded to IPFS:
+https://ipfs.io/ipfs/<ipfs-content-identifier>
+
+Your NFT was successfully minted in this block:
+https://explorer.shimmer.network/testnet/block/<blockId>
+
+
+
+Your file "weapon.jpg" was successfully uploaded to IPFS:
+https://ipfs.io/ipfs/<ipfs-content-identifier>
+
+Your NFT was successfully minted in this block:
+https://explorer.shimmer.network/testnet/block/<blockId>
+```
+
+### Ownership Hierarchy
+
+As described in the `mintNFT()` function and as you can see on the Shimmer explorer, all three NFTs have been minted on the same user address. Of course this is not a bad thing, but if you want your NFTs to reflect a certain hierarchy between them on-DLT, this doesn't really help:
+
+```text
+User Address
+├── avatar_nft
+├── mask_nft
+└── weapon_nft
+```
+
+In this example we want to show that both, the mask NFT and the weapon NFT are items that belong to your unique avatar NFT. In order to achieve a ownership hierarchy as presented below, you will send the mask NFT and the weapon NFT to the unique address of the avatar NFT:
+
+```text
+User Address
+└── avatar_nft
+    ├── mask_nft
+    └── weapon_nft
+```
+
+### Preparation
+
+In order to be able to send a NFT you will need its NFT ID and of course an address to send it to. In this case we will need the address of the avatar NFT and the IDs of both, the mask and weapon NFT. Since this tutorial here is based on the [Testnet Wallet Setup](https://wiki.iota.org/tutorials/wallet-setup) tutorial, you should already have a script to check your Stronghold account balance.
+
+Run the script `check-balance.js` and you should see three NFT IDs:
+
+```console
+node check-balance.js
+```
+
+Your console output should look similar to this:
+
+```console
+<accountName>'s Balance:
+{
+  baseCoin: { total: '1000000000', available: '1000000000' },
+  requiredStorageDeposit: '822000',
+  nativeTokens: [],
+  nfts: [
+    '<nft-id-avatar>',
+    '<nft-id-mask>',
+    '<nft-id-weapon>'
+  ],
+  aliases: [],
+  foundries: [],
+  potentiallyLockedOutputs: {}
+}
+```
+
+Now, one by one, copy the NFT IDs into the [Shimmer Testnet Explorer](https://explorer.shimmer.network/testnet/) and find out the one that belongs to the avatar NFT, by checking their immutable metadata. Before you move to the next section, note the NFT Address of the avatar NFT and also the NFT ID's of the mask and weapon NFT.
+
+```text
+avatar NFT Address:
+rms...
+
+mask NFT ID:
+0x...
+
+weapon NFT ID:
+0x...
+```
```

Comment:
This whole section kind of feels clumsy and hard to follow. Can't we just get this info directly from the response or from the explorer?

Reply:
Yes that should be possible by using the minting result and storing the IDs 🤔